build.gradle

```diff
@@ -1,10 +1,12 @@
 buildscript {
-    repositories {
+  ext.kotlin_version = '1.1.4-2'
+  repositories {
```

Comment:
Did the indentation of `repositories` change? Is it wrong now or was it wrong before?

Reply:
Ok let's not worry about it here